test_runner/src/test/kotlin/ftl/args/yml/ErrorParserTest.kt

```diff
@@ -63,7 +64,7 @@ At line: 23, column: 3
               "version" : "test"
             }
         """.trimIndent()
-        Assert.assertEquals(exceptedMessage, actualMessage)
+        assert(equalsIgnoreNewlineStyle(exceptedMessage, actualMessage))
```

Comment:
`assertEquals` is giving a more meaningful message. You have changed it to simple `assert` which not gives comprehensive information on failure. WDYF about changing function `equalsIgnoreNewlineStyle` to `assertEqualsIgnoreNewlineStyle`?

Reply:
Hmm Can do.